manifests/v/VOWSoft/iBackupBot/5.6.2/VOWSoft.iBackupBot.installer.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: VOWSoft.iBackupBot
+PackageVersion: 5.6.2
+Installers:
+- Architecture: x86
+  InstallerType: nullsoft

```

```suggestion
  InstallerType: nullsoft
  Scope: machine
```
`Scope: machine` is missing here.